### ISSUE TITLE
Release schedule

### ISSUE THREAD

**Issue by mkitti** [bytedeco/javacpp-presets#1351]
Is there a defined release schedule for JavaCPP and JavaCPP presets?

I was wondering if it would be worthwhile for me to backport the HDF5 1.14 update to JavaCPP 1.5.8 and release that via an independent maven project or if there was a release forthcoming.

**Comment by saudet** [comment]
If you're talking about 1.5.9, probably sooner rather than later, but I'll mark this as an enhancement for the general case that everyone is always asking about, but that no one is willing to do anything about.

**Comment by saudet** [comment]
Well, I suppose in your case since you're interested only with HDF5, like you say you could release it yourself.

**Comment by mkitti** [comment]
Right, but I think I would need to backport it to JavaCPP 1.5.8 in that case. That might be easy, but also not be necessary if JavaCPP 1.5.9 were to be released soon.
